tests/cli/test_cmds_status.py

```diff
@@ -584,7 +691,6 @@ def test_apply_args_filters_bad_service_name(mock_list_services, capfd):
     assert "Did you mean any of these?" in output
     assert "  fake_service" in output
 
-
```

Comment:
pep8 specifies 2 blank lines between top-level function definitions. Do you have the pre-commit hooks installed? They should fix this for you. Run `tox -e install-hooks`.